src/ceph-disk/ceph_disk/main.py

```diff
@@ -478,6 +479,20 @@ def command(arguments, **kwargs):
     return _bytes2str(out), _bytes2str(err), process.returncode
 
 
+def command_with_stdin(arguments, stdin):
+    LOG.info("Running command with stdin: " + " ".join(arguments))
+    process = subprocess.Popen(
+        arguments,
+        stdin=subprocess.PIPE,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE)
+    out, err = process.communicate(stdin)
+    LOG.debug(out)
+    LOG.error(err)
+    assert process.returncode == 0
```

Comment:
this should probably follow the same API of the other `command` function that raises when the exit code is not 0: it uses `SystemExit` while logging the error and the exit code